### PR TITLE
Create a README Markdown file

### PR DESCRIPTION
**Pull request by tiffany76**
This PR creates a README in the public repo. It replaces #3.

### CODE REVIEW

**Comment by tiffany76** [comment]
@ycombinator Please review at your convenience.

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Welcome 
+
+Welcome to the Courtgrabber Documentation public repository. These files have been formatted according to the MkDocs theme and are best viewed on an MkDocs site. We have not yet found a permanent hosting service, but we hope to make the move soon. In the meantime, you can view the files by downloading MkDocs, cloning this repo, and running the built-in MkDocs dev server to view the site locally.
```
Comment (ycombinator) [inline readme.md]: Nit: link the first reference to MkDocs to it's web site.

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+Run the following command from your command line to install MkDocs:
+
+`pip install mkdocs`
+
+If you encounter any issues, refer to the [MkDocs Installation Guide](https://www.mkdocs.org/user-guide/installation/) for more detailed instructions.
+
+## Creating your Courtgrabber project
+
+[Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repo to your local machine. Your project directory should contain the `mkdocs.yml` configuration file and a `docs` directory with four Markdown files and an `images` subdirectory.
```
Comment (ycombinator) [inline readme.md]: I'd avoid mentioning `four`.  If that number were to change in the future, this file will need to be updated as well.

File: readme.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Welcome 
+
+Welcome to the Courtgrabber Documentation public repository. These files have been formatted according to the MkDocs theme and are best viewed on an MkDocs site. We have not yet found a permanent hosting service, but we hope to make the move soon. In the meantime, you can view the files by downloading MkDocs, cloning this repo, and running the built-in MkDocs dev server to view the site locally.
+
+## Installing MkDocs
+
+Run the following command from your command line to install MkDocs:
+
+`pip install mkdocs`
```
Comment (ycombinator) [inline readme.md]: Nit: enclosing commands in single backticks is generally done when the command is inline with other non-monospaced text, like a regular sentence.  For commands (or code in general) that's in a standalone block, it's moe the norm to enclose it in triple backticks, with the triple backticks being on their own separate lines.

File: readme.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+## Creating your Courtgrabber project
+
+[Fork](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and [clone](https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository) this repo to your local machine. Your project directory should contain the `mkdocs.yml` configuration file and a `docs` directory with four Markdown files and an `images` subdirectory.
+
+## Viewing the Courtgrabber MkDocs site
+
+From your command line, navigate to your new project directory and then run the following command:
+
+`mkdocs serve`
```
Comment (ycombinator) [inline readme.md]: See other comment on single vs. triple backticks.

File: readme.md (hunk 1 above)
Comment (tiffany76) [reply readme.md]: Sure, will do.

File: readme.md (hunk 2 above)
Comment (tiffany76) [reply readme.md]: Okay. I changed it to "several" and will push the change shortly.

File: readme.md (hunk 3 above)
Comment (tiffany76) [reply readme.md]: Right. I forgot. Thanks.

File: readme.md (hunk 4 above)
Comment (tiffany76) [reply readme.md]: Right, thanks. Will do.

**Comment by tiffany76** [comment]
@ycombinator I've made the changes you suggested and pushed them. Thanks.

File: readme.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+## Viewing the Courtgrabber MkDocs site
+
+From your command line, navigate to your new project directory and then run the following command:
+
+```
+mkdocs serve
+```
+
+With the server running, open `http://127.0.0.1:8000/` in your browser to view Courtgrabber's documentation.
```
Comment (ycombinator) [inline readme.md]: Nit: You could consider leaving `http://127.0.0.1:8000/` unformatted so it renders as a link.  That makes it just a bit easier for a user reading this documenting while setting things up to visit the docs site in their web browser by simply clicking the link.

Comment (ycombinator) [inline readme.md]: You may want to include instructions on how to stop the server.

Comment (tiffany76) [reply readme.md]: Makes sense. I've made the change.

Comment (tiffany76) [reply readme.md]: Good point. Added.

**Review by ycombinator — APPROVED** [review]
LGTM.

**Pull request merged**